### PR TITLE
Refine the document

### PR DESCRIPTION
**Pull request by raphaellueckl**
If you think this is an improvement, please merge it. If you think some things need to change, but overall it is an improvement, please merge and add your improvements in a separate PR or add a commit directly in here.

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -15,18 +20,13 @@ Open your code editor of choice and edit your desired component/package in
...
 - We use [prettier](https://prettier.io/).
-  - Make sure you install eslint checker and prettier in your IDE
 
 ### Style Rules
 
 - We follow DRY, KISS, YAGNI principles when we write SCSS. This includes variables, mixins, etc.
 - We remove the linting of ordering and grouping of CSS properties. Only includes have to be at the beginning.
 - We won't have settings for vertical rhythm or global spacing.
 
-### JS Rules
```
Comment (LucaMele) [inline CONTRIBUTION.md]: This is still valid rule, even with prettier

Comment (raphaellueckl) [reply CONTRIBUTION.md]: but we removed some of them. the built ins being the most obvious ones.

Comment (LucaMele) [reply CONTRIBUTION.md]: We removed one or two and Prettier overrides the formatting rules. Still, Eslint is for code style, prettier for formatting and settings are alomst same as plib 1

Bildschirmfoto 2019-06-24 um 09 33 54

Comment (LucaMele) [reply CONTRIBUTION.md]: @raphaellueckl maybe update the text like so:

We use the plibv1 eslint styling for code style, instead prettier takes care of all the formatting rules in the plib2

Comment (raphaellueckl) [reply CONTRIBUTION.md]: I reverted it. I think it's better to go with the former approach imho, but we can also use your sentence. I think mentioning the linting rules here is enough.

**Pull request merged**